build.zig

```diff
@@ -1,31 +1,41 @@
-const build_mod = @import("std").build;
-const Builder = build_mod.Builder;
-const pkgs = @import("deps.zig").pkgs;
+const std = @import("std");
 
-pub fn build(b: *Builder) void {
+pub fn build(b: *std.Build) void {
     const target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
 
-    const mode = b.standardReleaseOptions();
+    const exe = b.addExecutable(.{
+        .name = "blurhash-zig",
+        .root_source_file = .{ .path = "main.zig" },
+        .target = target,
+        .optimize = optimize,
+    });
+    b.installArtifact(exe);
+    const zigimg = b.dependency("zigimg", .{
+        .target = target,
+        .optimize = optimize,
+    });
+    exe.addModule("zigimg", zigimg.module("zigimg"));
```

Comment:
This, and the corresponding line for `unit_tests`, no longer compiles on the most recent dev release of Zig (`0.12.0-dev.2543+9eda6ccef` as I'm writing this) unfortunately. I guess these days it ought to look like:
```
exe.root_module.addImport("zigimg", zigimg.module("zigimg"));
```

I was able to get it to run with the latest version by making those changes, and by also pinning zigimg to commit `61862e6a37e70551cccd4113bf987eaf93b67601` (also the latest one as of right now)

I'm thinking we can go ahead by either
- Making the above changes so that this commit can run with the latest Zig dev release
- You could provide me with the release of Zig that you used to make these changes so I can note it down and so that anyone who'd want to use this library knows which version of Zig to use

Let me know which of these makes the most sense to you 🙂 Ideally I would have liked to wait until Zig 0.12.0 actually lands but based on the latest roadmap presentation it seems like it might take some time

Reply:
I've upgraded Zig to `0.12.0-dev.2546+f2e249e92` and set it as a minimum version.
I agree it would be better to stick with release version, but recent `zigimg` already uses `0.12.0`.